### PR TITLE
Update to 1.12.2, use BiliLiveLib, add overlay.

### PR DESCRIPTION
**Pull request by cqjjjzr**
Signed-off-by: Charlie Jiang <cqjjjzr@126.com>

### CODE REVIEW

File: src/main/java/info/tritusk/projectbilistream/BiliLiveMonitor.java [hunk 1]
```diff
@@ -1,161 +1,67 @@
...
+import charlie.bililivelib.room.Room;
 import com.google.gson.Gson;
-import com.google.gson.internal.LinkedTreeMap;
 import net.minecraft.client.Minecraft;
-import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.util.text.TextComponentString;
-import net.minecraftforge.fml.common.FMLLog;
-import org.apache.commons.lang3.RandomUtils;
-import org.apache.logging.log4j.Level;
+import org.jetbrains.annotations.NotNull;
```
Comment (3TUSK) [inline src/main/java/info/tritusk/projectbilistream/BiliLiveMonitor.java:12]: `javax.annotation.Nonnull`?

File: src/main/java/info/tritusk/projectbilistream/Command.java [hunk 2]
```diff
@@ -24,11 +24,7 @@ public int getRequiredPermissionLevel() {
 
 	@Override
 	public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
-		BiliLiveMonitor.keepRunning = false;
-		while (BiliStreamMod.t.isAlive()) {
-		}
```
Comment (3TUSK) [inline src/main/java/info/tritusk/projectbilistream/Command.java:29]: 确定之前的线程可以直接丢弃？

File: src/main/java/info/tritusk/projectbilistream/Overlay.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiChat;
+import net.minecraft.client.gui.ScaledResolution;
+import net.minecraftforge.client.event.RenderGameOverlayEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+/**
+ * @author Charlie Jiang
+ * @since rv1
```
Comment (3TUSK) [inline src/main/java/info/tritusk/projectbilistream/Overlay.java]: `@since 0.0.1`

**Review by 3TUSK — CHANGES_REQUESTED** [review]
3 (very) minor things. Also, do we need... internationalization & localization?

File: src/main/java/info/tritusk/projectbilistream/BiliLiveMonitor.java (hunk 1 above)
Comment (cqjjjzr) [reply src/main/java/info/tritusk/projectbilistream/BiliLiveMonitor.java:12]: This is JetBrains annotations, from BiliLiveLib.

File: src/main/java/info/tritusk/projectbilistream/Command.java (hunk 2 above)
Comment (cqjjjzr) [reply src/main/java/info/tritusk/projectbilistream/Command.java:29]: Yes. Just do it.

**Comment by cqjjjzr** [comment]
I18n is unnecessary. I think only Chinese users will use this mod.

Also, 需要本地化的部分并不多。

**Review by 3TUSK — APPROVED** [review]

**Pull request merged**